chain/network/src/types.rs

```diff
@@ -1473,6 +1485,16 @@ pub enum NetworkClientResponses {
     DoesNotTrackShard,
     /// Ban peer for malicious behavior.
     Ban { ban_reason: ReasonForBan },
+
+    /// Sandbox controls
+    #[cfg(feature = "sandbox")]
+    SandboxResult(SandboxResponse),
```

Comment:
Position of `#[cfg(feature = "sandbox")]` in `NetworkClientMessages` and `NetworkClientResponses` is inconsistent. Let's keep it next to adversarial in both cases.

Reply:
Good catch